src/main/java/frc/robot/RobotContainer.java

```diff
@@ -97,14 +97,19 @@ public RobotContainer() {
     } else {
       intakeSubsystem = null;
     }
-    triggerSubsystem = new TriggerSubsystem();
-    if (Constants.enabledSubsystems.limeLightEnabled
-        && Constants.enabledSubsystems.drivetrainEnabled) {
-      visionSubsystem =
-          new LimelightSubsystem(drivetrain.getVisionMeasurementConsumer(), robotStateManager);
-    } else if (Constants.enabledSubsystems.photonEnabled
-        && Constants.enabledSubsystems.drivetrainEnabled) {
-      visionSubsystem = new PhotonSubsystem(drivetrain.getVisionMeasurementConsumer());
+    if (Constants.enabledSubsystems.triggerEnabled) {
```

Comment:
Not a required change, but I like to use ternaries for this type of thing.

Reply:
It should be resolved in the new commits